usaspending_api/etl/es_transaction_template.json

```diff
@@ -61,24 +75,28 @@
           "type": "text"
         },
         "award_description": {
-          "type": "text"
+          "type": "text",
+          "analyzer": "stemmer_analyzer"
         },
 
         "product_or_service_code": {
           "type": "text"
         },
         "product_or_service_description": {
-          "type": "text"
+          "type": "text",
+          "analyzer": "stemmer_analyzer"
         },
         "naics_code": {
           "type": "text"
         },
         "naics_description": {
-          "type": "text"
+          "type": "text",
+          "analyzer": "stemmer_analyzer"
         },
         "type_description": {
           "type": "text",
-          "fielddata": true
+          "fielddata": true,
+          "analyzer": "stemmer_analyzer"
```

Comment:
I don't believe this field needs to be changed. Example values are `NULL`, PO, DELIVERY ORDER,

Reply:
What branch did you base from? `type_description` is a keyword field now in dev. Please update.